tests/test_integration.py

```diff
@@ -286,7 +292,7 @@ def test_multiple_hosts(self):
                     output = output.getvalue()
                     lines = output.split('\n')
                     self.assertRegex(lines[3], r'^\| http://[\d\.:]+ .*\| NULL .*\| FALSE .*\| Server not available')
-                    self.assertRegex(lines[4], r'^\| http://[\d\.:]+. *\| crate .*\| TRUE .*\| OK')
+                    self.assertRegex(lines[4], r'^\| http://.*@?localhost:\d+ *\| crate .*\| TRUE .*\| OK')
```

Comment:
Is this always `localhost` because of testcontainers now?

Reply:
it really depends on two things:
1. if you pass `host` explicitly or rely on the default `None` value, when invoking `get_connection_url` method of the `CrateDBTestAdapter`
2. where the code is running, on the host or Docker-in-Docker, see the logic [here](https://github.com/testcontainers/testcontainers-python/blob/v3.7.1/testcontainers/core/container.py#L86). In our case, both for local development and in the CI pipelines, it will always be `localhost`

so, I'd say, it's safe to assume `localhost` won't break anything, unless `testcontainers` upstream or CI/CD pipelines in `crash` repo will be changed

Reply:
I see, thx!